apps/web/src/views/Launchpad/CreateCampaign.tsx

```diff
@@ -24,8 +27,53 @@ export const CreateCampaign: React.FC = () => {
 
   const [onPresentCreateModal] = useModal(<CreateModal formValues={formValues} />, true, true, 'tokenCreateModal')
 
+  const { address } = useAccount()
+  const [paid, setPaid] = useState<string | null>(null)
+  const [loading, setLoading] = useState<boolean>(true)
+  const [index, setIndex] = useState(0)
+  const user = useUser()
+
+  const handleClick = (newIndex) => {
+    setIndex(newIndex)
+
+    window.scrollTo(0, 0)
+  }
+
+  const getPaid = async () => {
```

Comment:
it is not whether the user has payed, but whether the User is KYCed. please rename getPaid to getKyced and setPaid, paid to setKyced, kyced.

Reply:
also make the kyced a bool as it only has 2 states. This is way more intuitive than having it as strings with REJECTED or NOT MINTED